ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/clientlibs/link-inspector-ui/js/console-ui.replace.js

```diff
@@ -110,8 +110,9 @@
         }
         if (data && data.updatedItemsCount) {
             var updatedItemsCount = data.updatedItemsCount;
-            var countMessage = `The number of updated items: <b>${updatedItemsCount}</b>`;
-            PROCESSING_SUCCESS_MSG = PROCESSING_SUCCESS_MSG.replace("%s", countMessage);
+            var updatedItems = data.updatedItems;
+            var updatedItemsMessage = `The number of updated items: <b>${updatedItemsCount}</b>${updatedItems}`;
+            PROCESSING_SUCCESS_MSG = PROCESSING_SUCCESS_MSG.replace("%s", updatedItemsMessage);
```

Comment:
Ughm, are you modifying a *constant* - ?) Since `PROCESSING_SUCCESS_MSG` is typed in upper-case, I believe it is assumed a constant, same as in Java

Reply:
Please correct this, the rest is allright